src/betterTableLayout.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @desc Combines mark and denominator into one cell
+ */
+const betterTableLayout = () => {
+    // fix table header
+    const row = $('#markbookTable tr:first');
+    const itemsHeader = row.find('td:nth-child(1)');
+    const marksHeader = row.find('td:nth-child(2)');
+    const dateHeader = row.find('td:nth-child(3)');
+    const weightHeader = row.find('td:nth-child(4)');
+
+    // reformat
+    row.empty();
+    row.append(itemsHeader, marksHeader, weightHeader, dateHeader);
+
+    // fix table content
+    $('#markbookTable tr:not(:first)').each(function () {
+        const row = $(this);
+        const name = row.find('td:nth-child(1)').clone(true, true);
+        const mark = row.find('td:nth-child(2)').clone(true, true);
+        const date = row.find('td:nth-child(3)').clone(true, true);
+        const weight = row.find('td:nth-child(4)').clone(true, true);
+        const denominator = $(this).find('td:nth-child(5)').clone(true, true);
+        
+        // combine mark and denominator
+        if (!settings.liveModification) {
+            let numerator = mark.text();
+            if (!isNaN(parseFloat(numerator)) && numerator !== '') {
+                numerator = +parseFloat(numerator).toFixed(2);
+            }
+            mark.html(`<span>${numerator}</span> <b>/</b> <span>${denominator.text()}</span>`);
+            mark.find('span:first').css({
+                'display': 'inline-block',
+                'margin': '0',
+                'text-align': 'right',
+                'width': '30pt'
+            });
+            mark.find('span:last').css({
+                'display': 'inline-block',
+                'margin': '0',
+                'text-align': 'left',
+                'width': '30pt'
+            });
+            // set mark column width
+            mark.css('width', '100px');
+        } else {
+            mark.append(' <b>/</b> ', denominator.find('input'));
+            // fix styling
+            const span = mark.find('span');
+            const input = mark.find('input:first');
+            if (span) {
+                span.css({
+                    'text-align': 'right',
+                    'width': '30pt'
+                });
+                span.bind('click', function () {
+                    input.show();
+                    input.focus();
+                    calculateMarks();
+                    $(this).remove();
+                });
+            }
+            mark.find('input:first').css({
+                'text-align': 'right',
+                'width': '30pt',
+                'padding-right': '5pt'
+            });
+            mark.find('input:last').css({
+                'text-align': 'left',
+                'width': '30pt',
+                'padding-left': '5pt'
+            });
+            // set mark column width
+            mark.css('width', '120px');
```

Comment:
Still way too wide, try this instead.

Using a 2px margin instead of spaces will solve the issue you highlighted earlier.

```suggestion
            mark.find('input:first').css({
                'text-align': 'right',
                'width': '30pt',
                'padding-right': '3pt',
                'margin-right': '2pt',
                'padding-left': '0pt'
            });
            mark.find('input:last').css({
                'text-align': 'left',
                'width': '30pt',
                'padding-left': '3pt',
                'margin-left': '2pt',
                'padding-right': '0pt'
            });
            // set mark column width
            mark.css('width', '100px');
```

Reply:
I'm also going to add padding on the other side because otherwise it isn't even.

![Screen Shot 2020-12-26 at 11 57 59 AM](https://user-images.githubusercontent.com/61663933/103157625-1c34d380-4772-11eb-9baf-0eba4d771c13.png) -> ![Screen Shot 2020-12-26 at 11 59 52 AM](https://user-images.githubusercontent.com/61663933/103157624-19d27980-4772-11eb-8bb3-6433b067f42c.png)